docker-hub/access-tokens.md

```diff
@@ -10,49 +10,52 @@ Using personal access tokens provides some advantages over a password:
 * You can investigate when an access token was used last, and disable or delete it if you find any suspicious activity.
 * When logged in with an access token, you can't perform any admin activity on the account, including changing the password.
 
+>**Warning**
+>
+> Treat access tokens like your password and keep them secret. Store your
+> tokens securely (for example, in a credential manager).
+{: .warning }
+
 Access tokens are also useful in building integrations, since you can issue
 multiple tokens &ndash; one for each integration &ndash; and revoke them at
 any time.
 
-> Note: If you have [two-factor authentication (2FA)](/docker-hub/2fa) enabled on your account, you must create at least one personal access token. Otherwise, you will be unable to log in to your account from the Docker CLI.
-{: .important }
+   >**Note**
+   >
+   > If you have [two-factor authentication (2FA)](/docker-hub/2fa) enabled on
+   > your account, you must create at least one personal access token. Otherwise,
+   > you will be unable to log in to your account from the Docker CLI.
+
 
 ## Create an access token
 
 You can create as many tokens as you need.
 
 1. Log in to [hub.docker.com](https://hub.docker.com).
```

Comment:
Could you add the target=_blank attribute here so the Hub page opens in a new tab?

Reply:
Updated